button.css

```diff
@@ -0,0 +1,88 @@
+:root {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+.button-container {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
+  display: inline-block;
+  font-weight: 800;
+  font-size: 1.8rem;
+  border: none;
+  border-radius: 1rem 1rem 1rem 1rem;
```

Comment:
you can just put border-radius: 1rem it will apply to all corners

Reply:
done with the changes u suggested